src/components/Builder/Index.jsx

```diff
@@ -36,80 +48,96 @@ const createPanes = (profileData, disableTabs) => [
   {
     key: PROJECTS_KEY,
     label: PROJECTS_LABEL,
-    children: <Project />,
+    children: <Project projectData={projectData} />,
     disabled: disableTabs
   },
   {
     key: EDUCATION_KEY,
     label: EDUCATION_LABEL,
-    children: <Education />,
+    children: <Education educationData={educationData} />,
     disabled: disableTabs
   },
   {
     key: EXPERIENCE_KEY,
     label: EXPERIENCE_LABEL,
-    children: <Experience />,
+    children: <Experience experienceData={experienceData} />,
     disabled: disableTabs
   }
 ];
 
-const achievement = (profileData, disableTabs) => ({
+const achievement = (achievementData, disableTabs) => ({
   key: ACHIEVEMENT_KEY,
   label: ACHIEVEMENT_LABEL,
-  children: <Achievement profileData={profileData} />,
+  children: <Achievement achievementData={achievementData} />,
   disabled: disableTabs
 });
 
-const certification = (profileData, disableTabs) => ({
+const certification = (certificationData, disableTabs) => ({
   key: CERTIFICATION_KEY,
-  label: <b>Certification</b>,
-  children: <Certification profileData={profileData} />,
+  label: CERTIFICATION_LABEL,
+  children: <Certification certificationData={certificationData} />,
   disabled: disableTabs
 });
 
 export const Editor = () => {
-  const { profile_id } = useParams();
-  const [items, setItems] = useState(createPanes(null, !profile_id));
-  const [profile, setProfile] = useState(PROFILES.internal);
   const resumeRef = useRef();
-  const [profileData, setProfileData] = useState(null);
+  const { profile_id } = useParams();
+  const [items, setItems] = useState(createPanes(null, null, null, null, true));
+  const [profiles, setProfiles] = useState(PROFILES.internal);
+  const [showCertification, setShowCertification] = useState(false);
+  const [showAchievement, setShowAchievement] = useState(false);
 
   const { data } = useGetBasicInfoQuery(profile_id ?? skipToken);
+  const { data: projectData } = useGetProjectQuery(profile_id ?? skipToken);
+  const { data: experienceData } = useGetExperiencesQuery(
+    profile_id ?? skipToken
+  );
+  const { data: educationData } = useGetEducationsQuery(
+    profile_id ?? skipToken
+  );
+  const { data: achievementData } = useGetAchievementsQuery(
+    profile_id ?? skipToken
+  );
+  const { data: certificationData } = useGetCertificatesQuery(
+    profile_id ?? skipToken
+  );
 
   useEffect(() => {
     if (profile_id) {
       if (data) {
-        setProfileData(data);
-        setItems(createPanes(data, false));
+        setItems(
+          createPanes(data, projectData, experienceData, educationData, false)
+        );
       }
     } else {
-      setItems(createPanes(null, true));
+      setItems(createPanes(null, null, null, null, true));
```

Comment:
pass a single object instead of sequence arguments

Reply:
I tried but i got error of null values. Can we discussed ?